tgui/packages/tgui/interfaces/TelecommsMonitor.js

```diff
@@ -0,0 +1,136 @@
+import { useBackend, useLocalState } from '../backend';
+import { Section, Stack, Input, Button, Table, LabeledList, NoticeBox } from '../components';
+import { Window } from '../layouts';
+
+const MachineScreen = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { network, machine } = data;
+  return (
+    <Stack fill vertical>
+      <Stack.Item>
+        <Section
+          title="Entity Information"
+          buttons={
+            <Button
+              content="Main Menu"
+              icon="home"
+              onClick={() => act('return_home')}
+            />
+          }>
+          <LabeledList>
+            <LabeledList.Item label="Network">{network}</LabeledList.Item>
+            <LabeledList.Item label="Network Entity">
+              {machine.name}
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+      </Stack.Item>
+      <Stack.Item grow>
+        <Section fill scrollable title="Linked Entities">
+          <Table>
+            <Table.Row header>
+              <Table.Cell>Address</Table.Cell>
+              <Table.Cell>Name</Table.Cell>
+              <Table.Cell>Machine</Table.Cell>
+            </Table.Row>
+            {machine.machinery?.map((m) => (
```

Comment:
Full name--`machine`

Reply:
I'm sorry, but I don't really understand what you're talking about